.github/workflows/alpine.yml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: mkdir build && cd build && cmake -G Ninja ../
       - name: Build
         run: cd build && cmake --build .
-      - name: Test
+      - name: Server Test
```

Comment:
Is it possible on alpine to start at least those tests that do not require gui?
Or do all require gui?

Reply:
The whole testing suite requires gui. Thats the vscode way

Reply:
OK. In the future, we should find a way to test it with theia & selenium